rubocop-compeon.gemspec

```diff
@@ -0,0 +1,27 @@
+require_relative 'lib/rubocop/compeon/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "rubocop-compeon"
+  spec.version       = RuboCop::Compeon::VERSION
+  spec.authors       = ["Timo Schilling"]
+  spec.email         = ["timo@schilling.io"]
```

Comment:
Nitpicking: probably the compeon address would be more suitable.

Reply:
Nope, it's my rubygems one, maybe we will release it there and not on GH packages.